js/ui/windowManager.js

```diff
@@ -1440,6 +1440,23 @@ const WindowManager = new Lang.Class({
                     this._checkDimming(parent);
         }));
 
+        let window = actor.meta_window;
```

Comment:
Nitpick: avoid redefining `window`, since it's a global variable in js. (Either pass `actor.meta_window` directly or use `win` or `metaWindow`.)